x-pack/plugins/observability_solution/slo/server/services/get_slos_overview.ts

```diff
@@ -131,10 +129,10 @@ export class GetSLOsOverview {
     const aggs = response.aggregations;
 
     return {
-      violated: aggs?.violated.doc_count ?? 0,
-      degrading: aggs?.degrading.doc_count ?? 0,
-      healthy: aggs?.healthy.doc_count ?? 0,
-      noData: aggs?.noData.doc_count ?? 0,
+      violated: aggs?.not_stale?.violated.doc_count ?? 0,
+      degrading: aggs?.not_stale?.degrading.doc_count ?? 0,
+      healthy: aggs?.not_stale?.healthy?.doc_count ?? 0,
+      noData: aggs?.not_stale?.noData.doc_count ?? 0,
       stale: aggs?.stale.doc_count ?? 0,
       worst: {
```

Comment:
Can you remove this, it is not used I just checked from the frontend, and it's an internal API. Also remove it from the schema: 
https://github.com/elastic/kibana/blob/main/x-pack/packages/kbn-slo-schema/src/rest_specs/routes/get_overview.ts#L21-L24